### PR TITLE
More polite error messages

### PR DESCRIPTION
**Pull request by jglick**
@reviewbybees

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/workflow/multibranch/SCMVar.java [hunk 1]
```diff
@@ -63,7 +64,7 @@
         Run<?,?> build = script.$build();
         // TODO some code overlap with SCMBinder.create, but not obvious how to factor out common parts
         if (!(build instanceof WorkflowRun)) {
-            throw new AbortException("not available outside a Pipeline build");
+            throw new AbortException("‘checkout scm’ is not available outside a Pipeline build");
```
Comment (abayer) [inline src/main/java/org/jenkinsci/plugins/workflow/multibranch/SCMVar.java]: :ant: Not a huge fan of the typeset quotes, but not a dealbreaker.

**Comment by abayer** [comment]
:bee: modulo my distaste for typeset quotes. =)

**Comment by jglick** [comment]
```
java.lang.AssertionError: expected:<[]> but was:<[org.jenkinsci.plugins.workflow.job.properties.PipelineTriggersJobProperty@25d0566a]>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at org.jenkinsci.plugins.workflow.multibranch.JobPropertyStepTest.configRoundTripBuildDiscarder(JobPropertyStepTest.java:85)
```

The PR builder seems to be mixing this up with #20 for reasons I cannot fathom.

**Comment by abayer** [comment]
Seen that sort of thing happen a couple times lately. Is odd. Close/reopen, I guess?

**Comment by jglick** [comment]
Looks like it was a problem with mismatched snapshots.

**Pull request merged**